rllib/env/single_agent_env_runner.py

```diff
@@ -95,6 +96,11 @@ def __init__(self, config: "AlgorithmConfig", **kwargs):
         self._ts_since_last_metrics: int = 0
         self._weights_seq_no: int = 0
 
+        # TODO (sven): This is a temporary solution. STATE_OUTs
```

Comment:
Temp. fix: We need the new connectors to make this work w/o having to keep `self._states` around here. The PRs for this are lined up and rely on this one here to be merged first.